body.go

```diff
@@ -1,5 +1,14 @@
 package postman
 
+//This const represents the language options->language in postman
```

Comment:
Generally, there is a space after the `//` :) Also, I'm not sure it's always necessary to specify `postman` everywhere as it is a Postman module.

```suggestion
// These constants represent the available raw languages.
```

Reply:
done.